<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>TriggerDetectorBitsInfoUi</class>
  <widget class="QWidget" name="TriggerDetectorBitsInfoUi" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>290</width>
        <height>190</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Form2</string>
    </property>
    <widget class="Q3Frame" name="frame7" >
      <property name="geometry" >
        <rect>
          <x>0</x>
          <y>0</y>
          <width>290</width>
          <height>190</height>
        </rect>
      </property>
      <property name="frameShape" >
        <enum>StyledPanel</enum>
      </property>
      <property name="frameShadow" >
        <enum>Raised</enum>
      </property>
      <layout class="QVBoxLayout" >
        <item>
          <widget class="Q3GroupBox" name="groupBox4" >
            <property name="title" >
              <string>TriggerBits (run/event)</string>
            </property>
            <layout class="QVBoxLayout" >
              <item>
                <widget class="QLineEdit" name="RunTrig" >
                  <property name="readOnly" >
                    <bool>true</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLineEdit" name="EventTrig" >
                  <property name="readOnly" >
                    <bool>true</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
        <item>
          <widget class="Q3GroupBox" name="groupBox5" >
            <property name="title" >
              <string>DetectorBits (run/event)</string>
            </property>
            <layout class="QVBoxLayout" >
              <item>
                <widget class="QLineEdit" name="RunDet" />
              </item>
              <item>
                <widget class="QLineEdit" name="EventDet" />
              </item>
            </layout>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
